index.d.ts

```diff
@@ -0,0 +1,22 @@
+interface CheckHandlerFunc {
+  (rule: any, value: any): string;
+}
+
+interface ValidateError {
+  code: string;
+  field: string;
```

Comment:
`field` can be `undefined`, add `?` modifier. others LGTM

Reply:
fixed